sources/packages/backend/apps/api/src/app.exception.filter.ts

```diff
@@ -6,14 +6,17 @@ import { LoggerService, InjectLogger } from "@sims/utilities/logger";
 @Catch()
 export class AppAllExceptionsFilter extends BaseExceptionFilter {
   catch(exception: unknown, host: ArgumentsHost) {
-    // TODO: Customize error response
     const ctx = host.switchToHttp();
     const request: Request = ctx.getRequest();
 
     // Logging Additional info
-    this.logger.error("Un-handle exception");
-    this.logger.error(`Request Path [${request.path}]`);
-    this.logger.error(`Exception Details: \n ***** \n\t${exception} \n *****`);
+    this.logger.error("Unhandled exception");
+    this.logger.error(`Request path [${request.path}]`);
+    this.logger.error(
+      `Exception details: \n ***** \n\t${
+        exception["response"] ? exception["response"]["message"] : exception
```

Comment:
Here we are assuming that if response is present, then `response.message` must be available. 

Possibly extend the condition 
```
exception["response"] &&  exception["response"]["message"]  ? 
exception["response"]["message"] : error;

```

Reply:
Using `exception["response"]["message"]` is not recommended. It is bypassing exactly what the `unknown` is trying to restrict and enforce.
Since this was not part of the PR I would recommend just doing a simple `JSON.stringify(exception)` (if possible).

Reply:
Do you prefer this way, @andrewsignori-aot ?
![image](https://user-images.githubusercontent.com/78114138/216209212-9c49dfb8-bc4e-4956-b2fb-f3cb90eee261.png)

Reply:
Short answer, yes 😉
This would be better because we would have access to the entire information and any piece of it can be valid for troubleshooting.
A middle-term solution would be figuring out if the `exception` has a type defined on Nestjs (or creating a type for it) that allows access(typed) to the most important information to avoid something like, what seems to be, repeated information shown in the above printscreen shared.
Since it was not initially part of this ticket, to keep it simple, I would go with the `JSON.stringify(exception)`.
@dheepak-aot thoughts?

Reply:
I agree.